src/sentry/incidents/utils/process_update_helpers.py

```diff
@@ -74,6 +75,92 @@ def get_aggregation_value_helper(subscription_update: QuerySubscriptionUpdate) -
     return aggregation_value
 
 
+def get_eap_aggregation_value(
+    entity_subscription: EntitySubscription,
+    subscription_update: QuerySubscriptionUpdate,
+    snuba_query: SnubaQuery,
+    project_ids: list[int],
+    organization_id: int,
+    start: datetime,
+    end: datetime,
+    alert_rule_id: int | None = None,
+) -> float | None:
+    try:
+        rpc_time_series_request = entity_subscription.build_rpc_request(
+            query=snuba_query.query,
+            project_ids=project_ids,
+            environment=snuba_query.environment,
+            params={
+                "organization_id": organization_id,
+                "project_id": project_ids,
+            },
+            referrer="subscription_processor.comparison_query",
+        )
+
+        rpc_time_series_request = add_start_end_conditions(rpc_time_series_request, start, end)
+
+        rpc_response = snuba_rpc.timeseries_rpc([rpc_time_series_request])[0]
+        if len(rpc_response.result_timeseries):
+            return rpc_response.result_timeseries[0].data_points[0].data
+
+    except Exception:
+        logger.exception(
+            "Failed to run RPC comparison query",
+            extra={
+                "alert_rule_id": alert_rule_id,
+                "subscription_id": subscription_update.get("subscription_id"),
+                "organization_id": organization_id,
+            },
+        )
+        return None
+
+
+def get_aggregation_value(
+    entity_subscription: EntitySubscription,
+    subscription_update: QuerySubscriptionUpdate,
+    snuba_query: SnubaQuery,
+    project_ids: list[int],
+    organization_id: int,
+    start: datetime,
+    end: datetime,
+    alert_rule_id: int | None = None,
+) -> float | None:
+    try:
+        # TODO: determine whether we need to include the subscription query_extra here
```

Comment:
unrelated to this pr -- but anyone have context over this comment?

Reply:
It's from an old Nathan PR https://github.com/getsentry/sentry/commit/028e31a2447524add5bbf447f98f53ae2e21d9e4#diff-d262481ba0aaae3473d14f62d3cbb554e099bda1093454b7935890139f3fe5a8 but I don't know the context, do you remember?